rasa_core_sdk/events.py

```diff
@@ -139,3 +139,12 @@ def AgentUttered(text=None, data=None, timestamp=None):
         "data": data,
         "timestamp": timestamp,
     }
+
+
```

Comment:
not that it makes a lot of sense, but also needs the actionexecutionfailed event

Reply:
done